doc/nims.rst

```diff
@@ -135,12 +135,6 @@ above example can be rewritten as:
   --define:foo
   --forceBuild
 
-**Note**: In general, the *define* switches can also be set in
```

Comment:
what about -d:strip, -d:lto, -d:lto_incremental ? refs https://github.com/nim-lang/Nim/pull/15614#issuecomment-856438314

Reply:
> -d:strip, -d:lto, -d:lto_incremental

I am not aware of those switches. Are you suggesting that we remove the mention for `-d:release` in the docs in that Note and replace that with these switches?

Reply:
yes, with the caveat that those flag names may be renamed in future

Reply:
> with the caveat that those flag names may be renamed in future

Do we need to mention that caveat? A lot of things get renamed and refactored in Nim.

Reply:
it's ok to not mention it